app-server/src/main/scala/rf/healthcheck/Healthcheck.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.rf.healthcheck
+
+import scala.concurrent.ExecutionContext
+
+import com.azavea.rf.utils.Database
+
+
+/**
+  * Individual service check for a component (database, cache, etc.)
+  * 
+  * @param service name of service that check is for
+  * @param status status of service (e.g. OK, UNHEALTHY)
+  */
+case class ServiceCheck(service: String, status: String)
+
+
+/**
+  * Overall healthcheck for Raster Foundry
+  * 
+  * @param status status of raster foundry (e.g. OK, UNHEALTHY)
+  * @param services list of individual service checks
+  * 
+  */
+case class HealthCheck(status: String, services: List[ServiceCheck])
```

Comment:
I read yesterday that the Twitter style guide prefers generic collections where possible i.e. `Seq` over `List` unless linked-list behavior is needed. I don't think I know enough about Scala yet to have strong preferences, so I guess this is more a question about what our starting point should be for Scala style.